docs/core/services/nodes-api.service.md

```diff
@@ -83,7 +83,10 @@ Accesses and manipulates ACS document nodes using their node IDs.
     -   _nodeId:_ `string`  - ID of the target node
     -   _nodeBody:_ `any`  - New data for the node
     -   _options:_ `any`  - Optional parameters supported by JS-API
-    -   **Returns** [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`MinimalNode`](https://github.com/Alfresco/alfresco-js-api/blob/master/src/alfresco-core-rest-api/docs/NodeMinimalEntry.md)`>` - Updated node information
+    -   **Returns** [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`MinimalNode`](https://github.com/Alfresco/alfresco-js-api/blob/master/src/alfresco-core-rest-api/docs/NodeMinimalEntry.md)`>` - Updated node information<br/>
+    Getting legal holds assigned to a node.
```

Comment:
Method declaration is missing here
<img width="530" alt="Zrzut ekranu 2024-06-12 o 08 23 11" src="https://github.com/Alfresco/alfresco-ng2-components/assets/113341662/ec5b8c1e-d77e-4f8c-9afa-1cedc76c7a18">

Reply:
updated, thanks!